bundles/org.openhab.ui.basic/src/main/java/org/openhab/ui/basic/internal/render/PageRenderer.java

```diff
@@ -303,6 +303,17 @@ public CharSequence renderSettings() throws RenderException {
                 localizeText("@text/ui.config.basic.adjustedColors.description"), "adjustedColors",
                 "openhab.ui.basic:adjustedColors", "enabled", "disabled", true, sb);
 
+        buttons = new StringBuilder();
+        buildButton("Default", "", buttons);
+        buildButton("Rounded", "rounded", buttons);
+        buildButton("Pill", "pill", buttons);
```

Comment:
Please localize the buttons label.

Reply:
I have no idea how I will translate "Pill" in French, I do not understand what it means in our context.

Reply:
I've localized them.

There's a picture at the _bottom_ of this page that shows a "Pill"
https://www.havardbrynjulfsen.design/writing/tips/pill-buttons

Posting some other photos here in case that link goes away
<img width="948" alt="image" src="https://github.com/user-attachments/assets/9e67ee76-f8ab-471e-a871-9600d70761a6" />

Reply:
Maybe I should call it "Pill-Shaped"? But that might make the translation (e.g. German) to be even longer. I tried google translate but I think a native speaker could pick a better translation.

Reply:
Ok, I see but I am not sure it will be clear for everybody.
Why not rather "Light rounded" for rounded and "More rounded" for pill ?

Reply:
And default could be replaced by "square" ?

Reply:
How about this:

<img width="450" alt="image" src="https://github.com/user-attachments/assets/f60c7d02-edb7-448b-8267-85dee4c7ab6e" />

Reply:
Do we keep "rounded" and "pill" as values for the local storage ?
Or do we align them a little with the new labels ?

Reply:
I've changed the value names to say "fully-rounded". Also found a bug when "Square" is selected, `ui-button-null` class was incorrectly set.